features/generate_fake_sample_data.feature

```diff
@@ -9,12 +9,18 @@ Feature: Generate Fake Sample Data
     And I am the journalist
 
   @javascript
-  Scenario:
+  Scenario: Generate Fake Sample Data
     When I visit the landing page
     And I click on "Sensors" in my dashboard
     And I click on "Temperature47" to see all the sensor readings
     And I click on "Generate Fake Sample Data"
     And I choose 10 random sensor readings with a value from 3°C to 20°C
     And I click on "Generate!"
-    Then I should see some entries in the sensor readings table
+    Then I should see some generated entries in the sensor readings table
     And this sensor should have 10 new sensor readings as fake data
+
+  Scenario: Distinguish Fake from Real Data
+    Given I have fake and real sensor readings for sensor "Temperature47"
+    When I see the page of this sensor
+    Then fake and real data are distinguishable
```

Comment:
@drjakob new behaviour, there aren't two databases but one column in sensor readings to distinguish fake from real data